### ISSUE TITLE
[HelpHub] Screencast

### ISSUE THREAD

**Issue by zzap** [WordPress/Documentation-Issue-Tracker#169]
Quick video overview of how to contribute this change (with English subtitles): https://wordpress.tv/2022/03/04/update-screenshots-in-wordpress-documentation/

Original article: https://wordpress.org/support/article/screencast-block/

## General
- [ ] Make sure all screenshots are relevant to the latest version 
- [ ] Make sure videos are up to date
- [ ] Add ALT tags for the images
- [ ] Make sure the headings are in sentence case
- [ ] Convert all reusable blocks to a ‘regular block’.
- [ ] Update the changelog at the end of the article

Issue migrated from Trello: https://trello.com/c/Ig6DrqSj/46-screencast

**Comment by samxmunoz** [comment]
Working on this tomorrow!!

**Comment by samxmunoz** [comment]
@zzap / @femkreations - it looks like Screencast.com is no more.

I even tried to use the URL shown in the original article but it won't embed.

**How would you like to proceed?**

**Comment by samxmunoz** [comment]
I submitted a Bug Report to Gutenberg RE: Screencast links no longer being embeddable:

https://github.com/WordPress/gutenberg/issues/46717 

CC: @zzap & @femkreations

**Comment by femkreations** [comment]
Yes the screencast block is not working just as @samxmunoz reported. The free version on [techsmith](https://www.techsmith.com/screencast.html) lets you share the screencast videos. 

Sample video to test -  https://app.screencast.com/oigpGQAplA5Yp

The GB issue created for this block in Dec has no progress. I think the embed URL has changed, (it is now https://app.screencast.com/something not sure if that's causing the block to not work.

The page is already there in [HelpHub](https://wordpress.org/documentation/article/screencast-block/) and I updated it to add info about creating a free Screencast account with links. But I cannot confirm the working of the block. 

Closing this issue for now
